mllp_http/http2mllp.py

```diff
@@ -1,17 +1,17 @@
+import functools
 import http.server
 import logging
 import socket
 import threading
 import time
-from .mllp import write_mllp
-from .net import read_socket_bytes
+from .mllp import read_mllp,write_mllp_socket
+from .net import read_real_socket_bytes
 
```

Comment:
Will you format with black?

(Or run with `make format` but that will also require some npm deps.)

Reply:
new commit formatted with black. sorry my vscode somehow doesn't auto format python.